.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    "env": {
+        "browser": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "indent": [
+            "error",
+            "tab"
```

Comment:
You're using spaces in all your existing files so this new rule would make your indentation inconsistent. I would prefer you keep it as spaces because it makes the diff far easier to read on GitHub (tabs render as 8 spaces).